src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const filteredStyles = styles.filter(
+    (styleDeclaration) => styleDeclaration.trim() !== '',
+  );
+
+  const styleObject = {};
+
+  filteredStyles.forEach((style) => {
+    const [property, value] = style.split(':').map((item) => item.trim());
```

Comment:
The destructuring assignment within the `forEach` callback is well done and makes the code more readable.